manager/testdata/new-blueprints/read-transform-blueprint.yaml

```diff
@@ -47,13 +49,15 @@ spec:
       selector: "mod: common-arrow-flight-read-service" # Selector of the dependency
       connection:
         service:
+          # name is the release name. It equals to M4DModule.spec.api.endpoint.host if such defined.
           name: common-arrow-flight-read-service
           assetID: m4d-notebook-sample/paysim-csv
           port: 8080
           interfaceDetails:
             protocol: arrow-flight
             dataformat: arrow
     service:
+      # name is the release name. It equals to M4DModule.spec.api.endpoint.host if such defined.
```

Comment:
What do you mean with release name here? Currently the release name is the name that is used to deploy the helm chart. This will differ from step to step and is the name e.g. in line 43. If two different applications configure a common read service they will have different release names but the service.name field here will have the same value.